Resources/Textures/Objects/Consumable/Drinks/scrantonspecialglass.rsi/meta.json

```
@@ -0,0 +1,34 @@
+{
+    "version": 1,
+    "size":
+    {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-NC-SA-3.0",
```

Consider licensing sprites with the [newer 4.0 version](https://creativecommons.org/version4/)

```suggestion
    "license": "CC-BY-NC-SA-4.0",
```